example/integration_test/cardano_ledger_address_derivation_tests.dart

```diff
@@ -169,13 +220,21 @@ void main() {
             'Derive stake address for account index ${testCase.accountIndex} and address index ${testCase.addressIndex} - isAppXS false',
             () async {
           if (isAppXS == false) {
-            final result = await cardanoApp.deriveStakingAddress(
-              device,
-              accountIndex: testCase.accountIndex,
-              addressIndex: testCase.addressIndex,
-              network: CardanoNetwork.mainnet(),
-            );
-            print('Derived address: $result');
+            try {
+              final result = await cardanoApp.deriveStakingAddress(
+                device,
+                accountIndex: testCase.accountIndex,
+                addressIndex: testCase.addressIndex,
+                network: CardanoNetwork.mainnet(),
+              );
+              print('Derived address: $result');
+            } catch (e) {
+              if (e is LedgerException) {
+                print('LedgerException caught: ${e.message}');
+              } else {
+                rethrow;
+              }
```

Comment:
Also, rethrow here should not be inside `else` block

Reply:
done